packages/playout-gateway/src/tsrHandler.ts

```diff
@@ -382,7 +383,7 @@ export class TSRHandler {
 		this.tsr.connectionManager.on('connectionEvent:debugState', (id, state) => {
 			const device = this._coreTsrHandlers[id]?._device
 
-			if (device?.debugState && this._coreHandler.logDebug) {
+			if (device && device.debugState && this._coreHandler.logDebug) {
```

Comment:
This change could be problematic, various linters will tell us to revert this.

Im curious why it needs to be split like this, and a comment stating this is a necessary change would be good to avoid it being reverted